crates/llms/Cargo.toml

```diff
@@ -22,14 +22,14 @@ tokio.workspace = true
 futures = { workspace = true }
 hf-hub = { version = "0.3.0", features = ["tokio"] }
 serde = { workspace = true, features = ["derive"] }
-tokenizers = { version = "0.20.0" }
+tokenizers = { version = "0.19.1" }
 tracing.workspace = true
 tracing-futures.workspace = true
 
 ## `candle` feature packages
-candle-core = { version = "0.5.0", optional = true }
```

Comment:
parts of mistral.rs fixes its candle-core to 0.5, but, even if true, I don't think they should clash.